Celeste.Mod.mm/Mod/Everest/Everest.Content.cs

```diff
@@ -798,9 +801,12 @@ public static void Add(string path, ModAsset metadata)
                     }
 
                 } else if (file.StartsWith("Ahorn/")) {
-                    // Special case: Fallback type for anything inside of the Ahorn folder.
+                    // Fallback types for anything inside of the Ahorn or Loenn folders.
                     // Will be ignored during collision checks.
```

Comment:
nitpick: this comment applies to both `else if` branches but is inside of one of them

Reply:
Yeah auto-formatter wasn't liking the comment position but I do like it better outside lol, fixed